README.md

```diff
@@ -64,6 +64,37 @@ Configuration is done via environment variables that can be different in differe
 
 7. Put the `DATABASE_URL` value (from setting up PostgreSQL locally) that works for you in it.
 
+8. Generating public/private key pairs for signing JWT
+	1. Go to: http://travistidwell.com/jsencrypt/demo/
+	2. Key Size: **512 bit**
+	3. Click on **Generate New Keys**
+	4. Open the `.env.example` file
+	5. Copy and paste the following lines from `.env.example` into your `.env` file: `PRIVATE_KEY` `PUBLIC_KEY` `ISSUER` `SUBJECT` `AUDIENCE` `EXPIRES_IN` `ALGORITHM`
+	6. Leave the values for `ISSUER` `SUBJECT` `AUDIENCE` `EXPIRES_IN` `ALGORITHM` as they are
+	7. Copy the private key from `iii` and paste it where the `PRIVATE_KEY=` in your `.env` file, do the same for the public key
+		a. When you paste in the keys, you will need to make adjustments to the strings, because they will be on separate lines
+
+
+```
+PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----
+MIICXAIBAAKBgQC
++9FiJCtty4mOb
+qOMWT2f6eNf+Jl
+5M4sQ6
+-----END RSA PRIVATE KEY-----"
+```
+
+Should be changed to
+
+
+```
+PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\nMIICXAIBAAKBgQC\n+9FiJCtty4mOb\nqOMWT2f6eNf+Jl\n5M4sQ6
+-----END RSA PRIVATE KEY-----"
+```
```

Comment:
I had to change it also include a newline character before the END RSA line for it to work, like this:
```
PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\nMIICXAIBAAKBgQC\n+9FiJCtty4mOb\nqOMWT2f6eNf+Jl\n5M4sQ6\n-----END RSA PRIVATE KEY-----"
```